### DISCUSSION TITLE
Difficulty change?

### DISCUSSION THREAD

**Discussion by xoonx** [SkriptLang/Skript#4864] [Suggestions] [answered]
Hello guys is there a way to change difficulty like:
If the servers diffuclty changes to hard, than its automaticly changes back to normal?
(bcs in modded server lucky block always changes the difficulty and we wanna keep it in normal xd)

if difficulty is hard
than
set difficulty to normal

or smtn like that, sorry i've never tried to code in my life tbh xD

**Comment by AyhamAl-Ali** [comment, marked as answer]
Please take a look at our [documentation](https://docs.skriptlang.org/)
Also you can ask any question and get help at [SkUnity discord help channels](https://discord.gg/xnS3JANY)

Here is the [difficulty expression](https://docs.skriptlang.org/expressions.html#ExprDifficulty) you're looking for.
